testing/python_testing/profile_it.py

```diff
@@ -0,0 +1,3 @@
+import cProfile
```

Comment:
If this file is going to be useful for future testing, might as well format it like the rest with a #!/usr/bin/env python3 and comment header/license